src/python/nxt_python.c

```diff
@@ -448,8 +449,22 @@ nxt_python_set_target(nxt_task_t *task, nxt_python_target_t *target,
                   callable, module_name);
         goto fail;
     }
+    
+    value = nxt_conf_get_object_member(conf, &factory_flag_str, NULL);
 
-    if (nxt_slow_path(PyCallable_Check(obj) == 0)) {
+
+    if(value != NULL && nxt_conf_get_boolean(value)) {
```

Comment:
Space between the `if` and the `(` please.

Also I'm unsure of putting  `nxt_conf_get_boolean(value)` in the `if` statement itself.

While this function does return a 0/1 (fake boolean, uint8_t), the name wouldn't necessarily indicate that as it's more of an action than a predicate.

But it does keep the patch smaller and I can live with it... hmm, what _would_ an alternative look like


```c
    ...
    nxt_bool_t        is_factory = 0;
    nxt_conf_value_t  *value;

    ...

    value = nxt_conf_get_object_member(conf, &factory_flag_str, NULL);
    if (value != NULL) {
        is_factory = nxt_conf_get_boolean(value);
    }

    if (is _factory) {
        // The rest of your new code...
    }
```

You know what, that isn't bad and reduces the if () to the core of its essence, so that would actually be my preference...